k8s/namespaces/wa/wa-task-batch-ad-hoc-delete-process-instances/wa-task-batch-ad-hoc-delete-process-instances.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: wa-task-batch-ad-hoc-delete-process-instances
+spec:
+  releaseName: wa-task-batch-ad-hoc-delete-process-instances
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/wa-task-batch-service
+  values:
+    job:
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/wa/task-batch-service:prod-2ba7463-20210702093219
+      schedule: "*/10 * * * *"
```

Comment:
Note: because we changed the job type to Job rather than CronJob
`schedule: "*/10 * * * *"` is redundant 

This pod will only run once :)

Reply:
done. Thanks